### PR TITLE
Marching percussion improvements

### PR DESCRIPTION
**Pull request by ZackTheCardshark**
Now that we have marching percussion playback, some attention is due to
the instruments.

* Genreless -> Common
* Keyboard shortcuts added
* Snares and cymbals 1-line -> 5-line staves
* Abbrev. names standardized
* “Drum1” -> “Drum 1” and similar
* Tenors straightened out in order of MIDI pitch
* Marching Percussion template added

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
CMake Error at share/templates/cmake_install.cmake:99 (FILE):
  file INSTALL cannot find
  "/home/travis/build/musescore/MuseScore/share/templates/07-Band/01-Concert_Band.mscz".


So you need to adjust share/templates/CMakeListe.txt to the renamed and added file

**Comment by ZackTheCardshark** [comment]
I'm beginning to like this Travis system. ;-) And thank you for the correction.

**Comment by ZackTheCardshark** [comment]
CMakeLists.txt fixed, and commits squashed.

File: share/templates/CMakeLists.txt [hunk 1]
```diff
@@ -76,8 +76,9 @@ install(FILES
       )
 
 install(FILES
-      07-Band/01-Concert_Band.mscz
-      DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}templates/07-Band
+      07-Band_&_Percussion/01-Concert_Band.mscz
+      07-Band_&_Percussion/02-Marching_Percussion.mscz
```
Comment (Jojo-Schmitz) [inline share/templates/CMakeLists.txt]: & in filenames are not allowed on Windows

File: share/templates/CMakeLists.txt [hunk 2]
```diff
@@ -76,8 +76,9 @@ install(FILES
...
+      DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}templates/07-Band_&_Percussion
```
Comment (Jojo-Schmitz) [inline share/templates/CMakeLists.txt]: & in filenames are not allowed on Windows

Comment (ZackTheCardshark) [inline share/templates/CMakeLists.txt]: I just went with "and."

**Comment by ZackTheCardshark** [comment]
All checks passed… finally. :-)

**Comment by lasconic** [comment]
I'm not familiar with the drumline tradition. Snares and cymbals 1-line -> 5-line staves, is that really standard?  The original instruments.xml from Mike Schorsch featured 1 line for these instruments. https://sites.google.com/site/mhsbulldogdrumline/composition

**Comment by ZackTheCardshark** [comment]
I'm basing this on:
1. ChurchOrganist's original marching percussion template (https://musescore.com/churchorganist/marching-percussion-template)
2. Images of scores from the internet, where it is pretty much universal (https://duckduckgo.com/?q=drumline+sheet+music&t=ffab&iax=1&ia=images)
3. The marching snare definition in 2.0.3 has notes on several different lines and spaces (not on one line)

**Pull request merged**